test/write_events.h

```diff
@@ -57,56 +66,56 @@ void write(std::string outfilename) {
 
     auto mcp1 = mcps.create();
     mcp1.setPDG( 2212 ) ;
-    mcp1.setMass( 0.938 ) ;
+    mcp1.setMass( 0.938d ) ;
```

Comment:
```suggestion
    mcp1.setMass( 0.938f ) ;
```
Floating point numbers are implicitly of type double. The `d` suffix is a gcc extension, thus the build failures with other compilers.